index.html

```diff
@@ -59,8 +59,23 @@
     </a>
 
     <!-- Hero -->
-    <section id="hero" class="section-pad-left">
-
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5-6, 2022</h3>
+            <h3>RIT Online Hackathon</h3>
+            <p>Apply today: </p>
+            <!-- Login and Signup buttons -->
+            <div id="hero-buttons">
+                <a id="login" href="https://apply.brickhack.io/">LOGIN</a>
+                <a id="register" href="https://apply.brickhack.io/users/sign_up">REGISTER</a>
```

Comment:
Do we want these to have `target="_blank"` to open in new tabs?

Reply:
That's a good question. Our goal for the site IS to move them to this page. Is there any point in the application process where they might simultaneously want to go back and look at something on the advertising site? Because in theory if this site does its job then they apply and don't really need it anymore.

Reply:
I'd use `target="_blank"` for anything that redirects to an external site, so I'd say yes? It can't hurt, at least.

Reply:
It's the same domain, so not external by definition.
My personal vote is to leave it, since our end goal is to send them there, they shouldn't really need to look at event info while filling out an application (because by that point they will have decided to apply), and because it's internal.

Reply:
Alright, that makes sense. I think I'm just in the habit of opening everything in a new tab, but I suppose I'd do that anyway regardless of what we set it to.